apps/mobile/src/components/HyloWebView/HyloWebView.js

```diff
@@ -77,6 +77,32 @@ const handledWebRoutesJavascriptCreator = loadedPath => allowRoutesParam => {
   `
 }
 
+// CSS to inject into WebView to fix scrollbar issues
+const injectedCSS = `
+  ::-webkit-scrollbar {
+    display: none !important;
+    width: 0 !important;
+    height: 0 !important;
+  }
+  
+  html, body {
+    overflow: hidden;
+    overflow-y: hidden;
+    overflow-x: hidden;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    width: 100vw !important;
+    position: fixed !important;
+    left: 0 !important;
+    right: 0 !important;
+    max-width: 100% !important;
+  }
+`
+
```

Comment:
Good you've got to the bottom of this! Tell me more on this part? I'm dubious that this part is WebView specific. WebKit specific for sure, which the first rule of course is explicitly. If I remember right I used to have a very similar set of rules set in Web as the issue of the scrollbar taking up space even when no overflow in Safari is a familiar issue to me. Either way, I'm not convinced that there is something WebView specific going on here. If so I'd like to understand what it is and why and to make a note of it here in a comment, and if not I'd then these should be applied to Web to address what I assume are WebKit / Safari issues on mobile devices and perhaps also on Desktop...

Reply:
Yeah, agreed that its not webview specific and could be applied elsewhere. I did try a version of applying this in web AuthRouter but that either didn't have the right impact (although I was using a much narrower set of style changes) or its in the wrong part of the component/styling hierarchy.

Somewhat terrified of adding a whole set of rules to the whole web app and then unintentionally breaking scrolling in a bunch of places. So felt safer to try at the 'inject into webview' level and assume Web styling is handling safari/browser differences on its own already.